web/static/js/components/action_item_generation_lower_third_content.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+
+import ActionItemSubmissionForm from "./action_item_submission_form"
+import StageProgressionButton from "./stage_progression_button"
+import stageConfigs from "../configs/stage_configs"
+
+import * as AppPropTypes from "../prop_types"
+
+const ActionItemGenerationLowerThirdContent = props => {
+  const { stage } = props
+  const stageConfig = stageConfigs[stage]
+
+  return (
+    <div className="ui stackable grid basic attached secondary center aligned segment">
+      <div className="thirteen wide column">
+        <ActionItemSubmissionForm {...props} />
```

Comment:
__Note to self:__ this markup is nearly identical to the markup seen in the [IdeaGenerationLowerThirdContent](https://github.com/stride-nyc/remote_retro/blob/c003622b7aced10d20e886f605e6b9c8894b3f43/web/static/js/components/idea_generation_lower_third_content.jsx). The only difference is that it renders the action item submission form, which in turn is nearly identical to the regular idea submission form. I get that there are slight differences between the two forms (assignee selection, rather than category selection), but can we refactor the existing form elements, containing markup, and related action creators and handlers to be more reusable, rather than introducing a whole new set of nearly identical modules, components, et cetera?